app/services/idv/vendor_validator.rb

```diff
@@ -25,5 +27,27 @@ def idv_agent
         vendor: (idv_session.vendor || idv_vendor.pick)
       )
     end
+
+    def result
+      @_result ||= try_submit
+    end
+
+    def try_agent_action
+      yield
+    rescue => err
+      err_msg = err.to_s
+      NewRelic::Agent.notice_error(VendorValidatorError.new(err_msg))
```

Comment:
Why bother wrapping the error message? I would think we can get better backtrace if we pass the actual error thrown by the proofer gem?